### ISSUE TITLE
 "https://registry.yarnpkg.com/picolors: Not found

### ISSUE THREAD

**Issue by mwoodpatrick** [samatechtw/nx-vue3-vite#47]
When running:

`npx nx g nx-vue3-vite:app vue-simple`

On Ubuntu 21.10 (WSL-2) I get the error:

`error An unexpected error occurred: "https://registry.yarnpkg.com/picolors: Not found".`

Does not look like that link exists, please advise

This is referenced in

`nx-vue3-vite/src/defaults.js`

Removing this from package.json does not appear to cause a problem and the generated app builds with one other issue being reported when running

` nx run vue-simple:lint`

Give output:

```
Linting "vue-simple"...

/mnt/wsl/projects/git/mwoodpatrick/westie/apps/vue-simple/src/app/components/widgets/Logo.vue
  1:1  error  Component name "Logo" should always be multi-word  vue/multi-word-component-names

✖ 1 problem (1 error, 0 warnings)

Lint errors found in the listed files
```.

Changing filename to Vue-Logo.vue fixes the problem

**Comment by sampullman** [comment]
That's odd, I will try to reproduce. We use PNPM or NPM for internal projects, so maybe there's an issue with Yarn? The package definitely exists: https://www.npmjs.com/package/picocolors

**Comment by westhechiang** [comment]
It's a typo in the package name. It's looking for `picolors` and the package you linked to is `picocolors`.

**Comment by sampullman** [comment]
Maybe I need glasses 😅 

Thanks for pointing that out, it's fixed in the `0.15.0` release.
